vendor/vendor-android/src/test/kotlin/com/malinskiy/marathon/android/executor/listeners/video/ScreenRecorderTestRunListenerTest.kt

```diff
@@ -64,8 +65,9 @@ class ScreenRecorderTestRunListenerTest {
                 serial("emulator-5554") {
                     boot()
                     shell(
-                        "screenrecord --size 720x1280 --bit-rate 1000000 --time-limit 180 /sdcard/com.example.Class-method1-${batch.id}.mp4",
-                        ""
+                        "screenrecord --size 720x1280 --bit-rate 1000000 --time-limit 180 /sdcard/com.example.Class-method1-${batch.id}-0.mp4",
```

Comment:
Is it possible to write tests checking that two or more chunks were created?

Reply:
In theory yes, but you'd have to wait more. Scheduler is not mocked here.